felix/fv/donottrack_test.go

```diff
@@ -161,8 +189,21 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ do-not-track policy tests;
 			// But external client should still be able to access it...
 			cc.ExpectSome(externalClient, hostW[1].Port(22))
 			cc.ExpectSome(externalClient, hostW[0].Port(22))
-			cc.CheckConnectivity()
 
+			if BPFMode() {
+				cc.Expect(None, tc.Felixes[0], hostW[1].Port(8055), ExpectWithIPVersion(6))
+				cc.Expect(None, tc.Felixes[1], hostW[0].Port(8055), ExpectWithIPVersion(6))
+				cc.Expect(Some, tc.Felixes[0], hostW[1].Port(2379), ExpectWithIPVersion(6))
+				cc.Expect(Some, tc.Felixes[1], hostW[0].Port(2379), ExpectWithIPVersion(6))
+				// Port 22 is inbound-only so it'll be blocked by the (lack of egress policy).
+				cc.Expect(None, tc.Felixes[0], hostW[1].Port(22), ExpectWithIPVersion(6))
+				cc.Expect(None, tc.Felixes[1], hostW[0].Port(22), ExpectWithIPVersion(6))
+				// But external client should still be able to access it...
+				cc.Expect(Some, externalClient, hostW[1].Port(22), ExpectWithIPVersion(6))
+				cc.Expect(Some, externalClient, hostW[0].Port(22), ExpectWithIPVersion(6))
```

Comment:
Could we put it in a function and use that function to setup the same expectations for either version and just pass in the extra `ExpectWithIPVersion` option? And in other similar places?